### PR TITLE
feat(ui): Add support for requesting collections in 4K

### LINKED ISSUE #639: 4k request support for "Request Collection"
**Is your feature request related to a problem? Please describe.**
Love the 4k request feature!!! Is it possible to support request collection as well with 4k request?

**Describe the solution you'd like**
I like to have the possibility to do a 4k request for a whole collection :)

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by TheCatLady**
#### Description

Adds support for requesting collections in 4K.

~~Also made one small/minor change to remove the icon from the regular "request" button dropdown children on movie/series detail pages, since there are no icons in other dropdowns (e.g., the "play" button dropdown children) and we should be consistent.  (If we want to add the icons to the request dropdowns, then we should add them everywhere else too!)~~

**Edit:** While testing, found that request buttons were being displayed on title cards even if users lacked the "Request" permission, so this is now fixed here.  Also, we were showing 4K status in some places even if a user lacked the relevant 4K request permissions; fixed that in this PR as well.

#### Screenshot (if UI related)

image
image

#### Todos

- [x] Successful build
- [x] Translation keys

#### Issues Fixed or Closed by this PR

- Fixes #639

### CODE REVIEW

File: src/components/RequestButton/index.tsx [hunk 1]
```diff
@@ -550,7 +550,6 @@ const RequestButton: React.FC<RequestButtonProps> = ({
                 onClick={button.action}
                 key={`request-option-${button.id}`}
               >
-                {button.svg}
```
Comment (sct) [inline src/components/RequestButton/index.tsx:553]: Why is this removed?

Comment (TheCatLady) [reply src/components/RequestButton/index.tsx:553]: As noted above:

> Also made one small/minor change to remove the icon from the regular "request" button dropdown children on movie/series detail pages, since there are no icons in other dropdowns (e.g., the "play" button dropdown children) and we should be consistent. (If we want to add the icons to the request dropdowns, then we should add them everywhere else too!)

Comment (TheCatLady) [reply src/components/RequestButton/index.tsx:553]: I'm totally cool with putting it back, but then I feel we should update the "play" button dropdown children to have icons as well.  Lmk what you think!

Comment (sct) [reply src/components/RequestButton/index.tsx:553]: I disagree with this change. Consistency doesn't directly always translate to clarity. The request dropdown has a lot of mixed options that do different things. The icons help quickly understand what you are clicking on at a glance.

Comment (sct) [reply src/components/RequestButton/index.tsx:553]: All of the actions for the `PlayButton` are "watching" something. They would all have the same icon. It would look silly.

Comment (TheCatLady) [reply src/components/RequestButton/index.tsx:553]: Hmmm okay.  I'll put it back 😄

**Review by sct — APPROVED** [review]
LGTM w/ tiny comment for future reference

File: src/components/CollectionDetails/index.tsx [hunk 2]
```diff
@@ -73,6 +117,7 @@ const CollectionDetails: React.FC<CollectionDetailsProps> = ({
           await axios.post<MediaRequest>('/api/v1/request', {
             mediaId: part.id,
             mediaType: 'movie',
+            is4k: is4k,
```
[the remark this answers is not in the document]
Comment (TheCatLady) [reply src/components/CollectionDetails/index.tsx:122]: I appreciate the nitpicking, and will change that!
